ci/README.md

```diff
@@ -1 +1,120 @@
-This is a WIP conversion of internal/mage CI targets to `dagger do` commands. It's not invoked by our CI yet.
+# CI
+
+This dagger module is used to define the CI for the dagger project itself,
+including building and releasing the CLI, engine and SDKs themselves.
+
+Available functionality:
+
+    $ dagger functions
+    Name     Description
+    cli      -
+    dev      Creates a dev container that has a running CLI connected to a dagger engine
+    docs     -
+    engine   -
+    sdk      -
+    test     -
+
+## Tests
+
+Run all tests:
+
+    $ dagger call --source=. test all
+
+Run a specific test (e.g. `TestModuleNamespacing`):
+
+    $ dagger call --source=. test custom --run="^TestModuleNamespacing" --pkg="./core/integration"
+
+## Dev environment
+
+Start a little dev shell with dagger-in-dagger:
+
+    $ dagger call --source=. dev
```

Comment:
Out of curiosity, is this a typical OS shell, or is this something much more specific to dagger, e.g. interactive dagger shell where you can run functions, etc.? 🤔